core/commonMain/src/kotlinx/io/bytes/BytesOutput.kt

```diff
@@ -0,0 +1,46 @@
+package kotlinx.io.bytes
+
+import kotlinx.io.*
+import kotlinx.io.buffer.*
+
+/**
+ * Create unlimited [Output] stored in memory.
+ * In advance to [Output] you can check [size] and create [BytesInput] with [createInput].
+ * [BytesOutput] isn't using any pools and shouldn't be closed.
+ */
+public class BytesOutput(
+    private val bufferSize: Int = DEFAULT_BUFFER_SIZE
```

Comment:
could you please elaborate why `BytesInput` constructor is internal, while this one is not?

Reply:
You can't create `BytesInput` from internal `Bytes` in public, and can create empty `BytesOutput` for write.